src/util.ts

```diff
@@ -56,7 +56,13 @@ export class DeadlineRunner<T> {
       this._reject(error);
   }
 
+  private _isDebuggerAttached() {
+    return process.env.NODE_OPTIONS?.includes('ms-vscode.js-debug') || /--debug|--inspect/.test(process.execArgv.join(' '));
```

Comment:
`return !!require('inspector').url()`

Reply:
For some reason I thought I tested it, but your solution works, which is even better, awesome.